src/libsync/propagateremotedelete.cpp

```diff
@@ -28,6 +28,8 @@ Q_LOGGING_CATEGORY(lcPropagateRemoteDelete, "nextcloud.sync.propagator.remotedel
 
 void PropagateRemoteDelete::start()
 {
+    qCInfo(lcPropagateRemoteDelete) << "Start propagate remote delete job";
```

Comment:
@FlexW 
Maybe, you can add a filename here too?

"Going to delete:" << filename;

Reply:
good idea :)